pkg/operator/starter.go

```diff
@@ -681,22 +680,9 @@ func prepareExternalOIDC(
 	informerFactories authenticationOperatorInformerFactories,
 ) ([]libraryapplyconfiguration.NamedRunOnce, []libraryapplyconfiguration.RunFunc, error) {
 
-	// By default, this will exit(0) if the featuregates change
```

Comment:
Let's keep this comment; it's not immediately obvious.

Reply:
Sure thing, re-added.